src/pages/ExerciseDetails/VariantCard/index.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import styles from './variant_card.module.css';
+
+export const VariantCard = () => {
+  return (
+    <div className={styles.card}>
```

Comment:
This is basically an OverviewCard, we should just use that component

Reply:
OK